### ISSUE TITLE
[Error]: Could not broadcast input array from shape

### ISSUE THREAD

**Issue by jooaobrum** [CosmiQ/solaris#410]
Hello, I've been working with Solaris lib and I've followed the Workshop Tutorial on Github for training a new pre-trained model, using my own dataset (RIO).

Unfortunately, my image size is 438 x 408, and it doesn't fit to the input size of my xdxd model. Then, following instructions from here, I add some zeros in my image using the trainning augmentation (PadIfNeed) to 480x480. All right, my model has trained for 3 epochs and everything looked fine...

Finally, I did a script to run inference on the new model, but I don't know why, this function causes an error due to my input shape and I don't know what to do, it's the same config file... In the image below you can check the error on my terminal 

![image](https://user-images.githubusercontent.com/54000549/94262076-3fe54500-ff09-11ea-8112-f20ffe6c1ad9.png)

And I will paste my config file (which has been successfully trained). Anybody can help me with this error?
##########################################################################################
model_name: xdxd_spacenet4

model_path:
train: true
infer: true

pretrained: true
nn_framework:  torch
batch_size: 2

data_specs:
  width: 480
  height:  480
  dtype: 
  image_type: zscore
  rescale: false
  rescale_minima: auto
  rescale_maxima: auto
  channels: 3
  label_type: mask
  is_categorical: false
  mask_channels: 1
  val_holdout_frac: 0.025
  data_workers:

training_data_csv: 'config/rio_train.csv'
validation_data_csv:
inference_data_csv: 'config/rio_test.csv'

training_augmentation:
  augmentations:
    PadIfNeeded:
      min_height: 480  # or whatever's the next biggest divisible-by-32 number
      min_width: 480
    HorizontalFlip:
      p: 0.5
    RandomRotate90:
      p: 0.5
    RandomCrop:
      height: 480
      width: 480
      p: 1.0
    Normalize:
      mean:
        - 0.3212980202822025
        - 0.34481500244167196
        - 0.25661383445115876
      std:
        - 0.16574531161473616
        - 0.14980481104147172
        - 0.14049872236919067
      max_pixel_value: 255.0
      p: 1.0
  p: 1.0
  shuffle: true
validation_augmentation:
  augmentations:
    PadIfNeeded:
      min_height: 480  # or whatever's the next biggest divisible-by-32 number
      min_width: 480
    Normalize:
      mean:
        - 0.3212980202822025
        - 0.34481500244167196
        - 0.25661383445115876
      std:
        - 0.16574531161473616
        - 0.14980481104147172
        - 0.14049872236919067
      max_pixel_value: 255.0
      p: 1.0
  p: 1.0
inference_augmentation:
  augmentations:
    PadIfNeeded:
      min_height: 480  # or whatever's the next biggest divisible-by-32 number
      min_width: 480
      p : 1.0
    Normalize:
      mean:
        - 0.3212980202822025
        - 0.34481500244167196
        - 0.25661383445115876
      std:
        - 0.16574531161473616
        - 0.14980481104147172
        - 0.14049872236919067
      max_pixel_value: 255.0
      p: 1.0
  p: 1.0
training:
  epochs:  3
  steps_per_epoch:
  optimizer: Adam
  lr: 1e-5
  opt_args:
  loss:
    bcewithlogits:
    jaccard:
  loss_weights:
    bcewithlogits: 10
    jaccard: 2.5
  metrics:
    training:
    validation:
  checkpoint_frequency: 10
  callbacks:
    model_checkpoint:
      filepath: 'models/xdxd_rio_best.pth'
      monitor: val_loss
  model_dest_path: 'models/xdxd_rio_final.pth'
  verbose: true

inference:
  window_step_size_x:
  window_step_size_y:
  output_dir: 'inference_out/'
